src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,251 @@
+    private static int totalTasks = 0;
+
+    private static final String home = System.getProperty("user.home");
+    private static final String dirPath = home + File.separator + "Documents" + File.separator + "CS2113T";
+    private static final String filePath = home + File.separator + "Documents" + File.separator + "CS2113T"
+            + File.separator + "tasks.txt";
+
+    private static void loadFile() {
+        if(!new File(dirPath).exists()) {
+            System.out.println("Folder does not exist yet.");
+            File dir = new File(dirPath);
+            if(dir.mkdir()) {
+                System.out.println("New folder is created at this location: " + dirPath);
+            } else {
+                System.out.println("Oops! There's some problem in creating new folder.");
+            }
+        }
+        try {
+            readFileContents();
+        } catch (FileNotFoundException e) {
+            System.out.println("File is not found.");
+        }
+    }
+
+    private static void readFileContents() throws FileNotFoundException {
+        File f = new File(filePath);
+        try {
+            if(f.createNewFile()) {
+                System.out.println("New file is created in the folder for storing task list.");
+            }
+        } catch(IOException e) {
+            System.out.println("An error has occurred.");
+        }
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            String thisLine = s.nextLine();
+            String[] words = thisLine.split("\\|", 4);
+            switch(words[0].trim()) {
+            case "T":
+                tasks.add(new Todo(words[2].trim()));
+                break;
+            case "E":
+                tasks.add(new Event(words[2].trim(), words[3].trim()));
+                break;
+            case "D":
+                tasks.add(new Deadline(words[2].trim(), words[3].trim()));
+                break;
+            default:
+                System.out.println("This line is not added to list:" + thisLine);
+            }
+            if(Integer.parseInt(words[1].trim())==1) {
+                tasks.get(totalTasks).markAsDone();
+            }
+            totalTasks++;
+        }
+    }
+
+    private static void saveTaskList() throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        for(int i = 0; i< totalTasks; ++i) {
+            fw.write(tasks.get(i).storeString() + "\n");
+        }
+        fw.close();
+    }
+
+    private static void addTodo(String task) {
+        tasks.add(new Todo(task));
+    }
+
+    private static void addEvent(String task) {
+        String[] words = task.split("/at");
+        tasks.add(new Event(words[0], words[1]));
+    }
+
+    private static void addDeadline(String task) {
+        String[] words = task.split("/by");
+        tasks.add(new Deadline(words[0], words[1]));
+    }
+
+    private static void updateDone(int updateNumber) {
```

Comment:
The naming of the variable `updateNumber` seems to suggest it is a function instead. Perhaps `numberToUpdate` might be a better choice?